addons/fetchmail/models/fetchmail.py

```diff
@@ -130,7 +130,7 @@ def button_confirm_login(self):
                 connection = server.connect()
                 server.write({'state': 'done'})
             except Exception as err:
-                _logger.info("Failed to connect to %s server %s.", server.type, server.name, exc_info=True)
+                _logger.error("Failed to connect to %s server %s.", server.type, server.name, exc_info=True)
                 raise UserError(_("Connection test failed: %s") % tools.ustr(err))
```

Comment:
```suggestion
                raise RuntimeError(_("Connection test failed: %s") % tools.ustr(err))
```

For consistency we should probably also change the class of this exception